src/main/java/pl/piekoszek/gorskimatches/equation/EquationRandomizer.java

```diff
@@ -7,14 +7,23 @@
 @Component
 public class EquationRandomizer {
 
-    private final QuizzesGenerator equationGenerator;
+    private final QuizzesGenerator quizzesGenerator;
 
-    public EquationRandomizer(QuizzesGenerator equationGenerator) {
-        this.equationGenerator = equationGenerator;
+    public EquationRandomizer(QuizzesGenerator quizzesGenerator) {
+        this.quizzesGenerator = quizzesGenerator;
     }
 
     public String randomEquation() {
-        var quizzes = new ArrayList<>(equationGenerator.getAllSolutionsByQuiz().keySet());
+        var quizzes = new ArrayList<>(quizzesGenerator.getAllSolutionsByQuiz().keySet());
```

Comment:
I'd suggest creating this array in constructor once, instead of doing it every time new quiz is fetched